css/styles.css

```diff
@@ -80,15 +80,33 @@ li {
   text-align: center;
   padding: 16px;
   text-decoration: none;
+  flex-wrap: wrap;
 }
 
 .bookmark {
   background-color: azure;
   height: 40px;
   width: 30px;
-  border: 1 solid rgba(0, 204, 255);
+  border: 3px solid#7788ff;
   border-radius: 0 0 20px 20px;
   position: absolute;
   top: -10px;
   right: 40px;
 }
+
+footer {
+  background-color: #7788ff;
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 80px;
+  z-index: 1;
+}
```

Comment:
LGTM, Captain 👩🏼‍✈️

ein kleiner Rechtschreibfehler in deiner ersten Frage: CaptAIn :)